lib/screens/settings/settings_screen.dart

```diff
@@ -0,0 +1,207 @@
+import 'package:at_backupkey_flutter/widgets/backup_key_widget.dart';
+import 'package:at_client_mobile/at_client_mobile.dart';
+import 'package:at_common_flutter/services/size_config.dart';
+import 'package:atsign_atmosphere_pro/screens/settings/widgets/settings_buttons.dart';
+import 'package:atsign_atmosphere_pro/utils/colors.dart';
+import 'package:flutter/material.dart';
+import 'package:package_info_plus/package_info_plus.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+import '../../routes/route_names.dart';
```

Comment:
@Chandram-Dutta The changes look good, but it would be great if we use the same import conventions throughout.

Can you please change all the imports to use `import 'package:atsign_atmosphere_pro/.....` this pattern.

Reply:
Will do it today